packages/typespec-test/test/NetworkAnalytics.Management/generated/typespec-ts/review/arm-networkanalytics.api.md

```diff
@@ -298,7 +332,17 @@ export interface DataTypeUpdateProperties {
 }
 
 // @public
-export type DefaultAction = string;
+export type DefaultAction = "Allow" | "Deny";
+
+// @public
+export enum DefaultActionKnownValues {
+    Allow = "\"Allow\"",
+    Deny = "\"Deny\""
+}
+
+// @public (undocumented)
+export interface DeleteDataRequest {
```

Comment:
this seems to be a new model. not sure where it comes from

Reply:
so is this expected?

Reply:
same reason for anonymous empty model.